db/schema.rb

```diff
@@ -27,12 +27,19 @@
     t.string "filename", null: false
     t.string "content_type"
     t.text "metadata"
-    t.bigint "byte_size", null: false
+    t.integer "byte_size", null: false
```

Comment:
I wonder how the type changed without a migration. But it’s an unused table, so… 🤷🏼